packages/grafana-ui/src/components/Layout/Grid/GridItem.tsx

```diff
@@ -0,0 +1,58 @@
+import { css } from '@emotion/css';
+import React, { useCallback } from 'react';
+
+import { useStyles2 } from '../../../themes';
+
+export interface GridItemProps {
+  children: NonNullable<React.ReactNode>;
+  /** If false will the grid item inherits the display of the parent. 
+    If true, the display with 'contents' as value. See info: https://developer.mozilla.org/en-US/docs/Web/CSS/display#display_contents*/
+  displayContents?: boolean;
+  /** Specifies the column where the grid item starts within the grid */
+  columnStart?: number;
+  /** Specifies the  column where the grid item ends. 
+    If its value is span <number> the item spans across the provider number of columns*/
```

Comment:
```suggestion
  /** Specifies the column where the grid item ends.
    If its value is `span ${number}`, the item spans across the provided number of columns */
```
Using `<number>` here was throwing an error in console, since apparently the storybook was interpreting it as a `<number>` HTML tag, which doesn't exist.